include/grpc/event_engine/event_engine.h

```diff
@@ -261,6 +261,11 @@ class EventEngine {
     struct LookupTaskHandle {
       intptr_t key[2];
     };
+    /// Optional configuration for DNSResolvers.
+    struct ResolverOptions {
+      /// Override the DNSResolver's default authority.
+      std::string authority;
```

Comment:
Using the term "authority" here seems weird, because this API is not expressed in terms of "dns:" URIs.  Instead, I suggest calling this "dns_server".

Reply:
Done